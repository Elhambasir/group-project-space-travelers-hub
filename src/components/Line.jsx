const Line = () => <div className="Line" />;

export default Line;
