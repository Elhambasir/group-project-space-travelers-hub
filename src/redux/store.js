// eslint-disable-next-line import/no-extraneous-dependencies
import { configureStore } from '@reduxjs/toolkit';
import MissionReducer from './mission/missionSlice';

// eslint-disable-next-line import/prefer-default-export
export const store = configureStore({
  reducer: {
    missions: MissionReducer,
  },
});
