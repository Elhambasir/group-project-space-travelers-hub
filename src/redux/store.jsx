// eslint-disable-next-line import/no-extraneous-dependencies
import { configureStore } from '@reduxjs/toolkit';
import rocketsReducer from './Rockets/rocketsSlice';
import MissionReducer from './mission/missionSlice';

const store = configureStore({
  reducer: {
    rockets: rocketsReducer,
    missions: MissionReducer,
  },
});

export default store;
