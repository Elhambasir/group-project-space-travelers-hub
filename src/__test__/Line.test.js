import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Line from '../components/Line';
import store from '../redux/store';

describe('Component render testing', () => {
  it('Line page renders correctly', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Line />
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
