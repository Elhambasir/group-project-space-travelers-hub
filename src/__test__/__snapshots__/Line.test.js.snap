// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component render testing Line page renders correctly 1`] = `
<DocumentFragment>
  <div
    class="Line"
  />
</DocumentFragment>
`;
